lib/active_model/one_time_password.rb

```diff
@@ -96,11 +96,19 @@ def otp_column=(attr)
       end
 
       def otp_counter
-        self.public_send(self.class.otp_counter_column_name)
+        if self.class.otp_counter_column_name != 'otp_counter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.